mailpoet/assets/js/src/automation/sections/build-your-own-section.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { MailPoet } from '../../mailpoet';
+
+export function BuildYourOwnSection(): JSX.Element {
```

Comment:
This seems to be somewhat misaligned in my browser (Chrome, macOS) — the underline as well as heading vs. details text:

<img width="539" alt="Screenshot 2022-10-27 at 9 45 33" src="https://user-images.githubusercontent.com/141436/198222264-26f0ef60-d583-4830-8f9d-1f7b4982748b.png">



Specs:
<img width="734" alt="Screenshot 2022-10-27 at 9 44 36" src="https://user-images.githubusercontent.com/141436/198222064-c41faf0b-e2e1-4bbb-9eae-60c8bd5978e1.png">

Reply:
Also the spacing below the item here seems too large:

<img width="473" alt="Screenshot 2022-10-27 at 9 51 02" src="https://user-images.githubusercontent.com/141436/198223386-8a9a304b-cf5e-4819-a480-e80763b1a12a.png">

Reply:
58e9532e4

Reply:
After rebase: [be69f35](https://github.com/mailpoet/mailpoet/pull/4461/commits/be69f355fd253c86da311b916b8efd26566d950d)